src/GuestPages/RSVP.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Status from './Status';
+import NewRsvp from './NewRSVP';
+
+const RsvpForm = () => (
+  <Container fluid>
+    <Container fluid>
+      <div>Side Image??</div>
+      <div>Couple1 Name</div>
+      <div>&</div>
+      <div>Couple2 Name</div>
+      <div>invite you to their special day</div>
+      <div>Date</div>
+      <div>Time</div>
+      <div>Location</div>
+      <div>RSVP by Deadline</div>
```

Comment:
Since all of these elements will appear at the top of the Invite Form and appear to be centered/aligned similarly, could all these elements be contained in one div and each element be assigned appropriate attributes from there? Example would be Couple1 Name & Couple2 Name all being in one `h1` tag. Using specific tags for each of these pieces of data may also help with accessibility.

Reply:
Will also address this in my refactor!